start python run.py debug
timeout 5
python36 -m unittest discover